static/bidder-info/adsinteractive.yaml

```diff
@@ -8,4 +8,9 @@ capabilities:
       - banner
   site:
     mediaTypes:
-      - banner
\ No newline at end of file
+      - banner
+userSync:
+  redirect:
+    url: "http://pb.adsinteractive.com/getuid?{{.RedirectURL}}"
+    userMacro: "$AUID"
```

Comment:
This redirect URL doesn't seem to be redirecting to redirect URL and setting UID macro. Could you please check?
Here is the URL that I tried - 
```
http://pb.adsinteractive.com/getuid?https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dadsinteractive%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Di%26uid%3D%24AUID
```

Reply:
@balintvargha The URL is still not redirecting and adding the UID macro.

Reply:
@balintvargha `$AUID` macro is not replaced with uuid

<img width="1280" alt="Screenshot 2023-04-11 at 10 18 08 PM" src="https://user-images.githubusercontent.com/24757781/231233509-81dfe6b7-24c1-4816-8159-1da0871ae238.png">

Reply:
$AUID macro is now being replaced by uuid, please check now.

Reply:
endpoint is working. Thanks @balintvargha for following up on this

Reply:
look good now. thanks for being patient on this one!

Reply:
Thank you @onkarvhanumante and @gargcreation1992 for reviewing, also for your patience as well